src/CartLatticeLauncher.hpp.Rt

```diff
@@ -255,9 +255,9 @@ public:
 
   CudaDeviceFunction void Execute() const {
     using LA = CartLatticeAccessAll;
-    int x = CudaBlock.x + small.dx;
-    int y = CudaBlock.y + small.dy;
-    int z = CudaBlock.z + small.dz;
+    int x = (CudaBlock.x + small.dx) %container.nx;
```

Comment:
Why did this appear here? x,y,z should always be in container's region, and modulu ("%") is very expensive and generally not very reliable.